datadog_lambda/wrapper.py

```diff
@@ -190,6 +192,9 @@ def _after(self, event, context):
             status_code = extract_http_status_code_tag(self.trigger_tags, self.response)
             if status_code:
                 self.trigger_tags["http.status_code"] = status_code
+                if not self.already_submitted_errors_metric_before:
+                    if len(status_code) == 3 and status_code.startswith("5"):
+                        submit_errors_metric(context)
```

Comment:
You also need to mark `self.span` as error, otherwise the trace and metric won't match.

Reply:
Please let me know if I miss anything to set `self.span` as error. Thanks!
https://github.com/DataDog/datadog-lambda-python/pull/229/commits/ad09cc90874f0d911637a939cd295968ff308ea8#diff-5efc779a7e3c73ebd812eb62ff5dce68db1e0ed9ef9765bb076638733e8c5ae6R195-R196